app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,87 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* universal */
+$parchment: #EBE5D5;
+$dark_blue: #112230;
+$light_blue: #68b7e1;
+$ocean_green: #70A58B;
+$dark_ocean_green: #649d81;
+$grey: #3D4B50;
+$sand: #B9A167;
+$crab_red: #E24D34;
+$shrimp_pink: #E49E78;
+$dark_shrimp_pink: #e19268;
+$clam_tan: #D7CAB7;
+$dark_clam_tan: #c2ae91;
+$dark_dark_clam_tan: #b49b78;
+$seafood_parchment: #E8E1D4;
+
+body {
+    background-color: $parchment;
+}
+
+h1, h2 {
+    color: $ocean_green;
+}
+
+#big-title {
+    font-color: $ocean_green;
+}
+
+a:link, a:visited, a:active {
+  color: $dark_dark_clam_tan;
+  text-decoration: none;
+}
+
+.alert-success {
+    color: white;
+    background-color: $ocean_green;
+}
+
+.alert-danger {
+    color: white;
+    background-color: $shrimp_pink;
+}
+
+a:hover, a:active {
+    color: $dark_clam_tan;
+}
+
+#sign-in {
+    color: white
+}
+
+#sign-out {
+    color: white
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Declaration should be terminated by a semicolon